en/docs/setup/token-persistence.md

```diff
@@ -29,28 +29,35 @@ The flow of synchronous token persistence is as follows:
     using the same thread. Once it is persisted, the new token is
     returned to the client.
     
-!!! note "Enable synchronous token persistence"
-    Navigate to file `<IS_HOME>/repository/conf/deployment.toml` and add the following configuration.
-    
+!!! note "Synchronous token persistence configurations" 
+    By default
+    synchronous token persistence is enabled in WSO2 Identity Server 5.9.0
+    onwards. To indicate how many times to retry in the event of a
+    `CONN_APP_KEY` violation when storing the access token, Navigate to file
```

Comment:
please start navigate with simple case. navigate to the <
<IS_HOME>/repository/conf/deployment.toml/deployment.toml file